src/main/java/net/openhft/chronicle/network/TcpEventHandler.java

```diff
@@ -671,11 +670,7 @@ public boolean action() throws InvalidEventHandlerException {
                             .append(" ")
                             .append(msg.logType.label())
                             .append(" took ")
-                            .append(msg.elapsedNs / 1000)
-                            .append(" us, CPU: ")
-                            .append(Affinity.getCpu())
-                            .append(", affinity ")
-                            .append(Affinity.getAffinity());
+                            .append(msg.elapsedNs / 1000);
```

Comment:
I think we should
1. create an issue for this
2. add the CPU to the `ThreadLogTypeElapsedRecord` so we can log the correct one

Reply:
I'm guessing we don't need to add Affinity.getAffinity() to the record because everyone should get the same values back from that?

Reply:
Actually the return value is thread-specific `Returns: returns affinity mask for current thread, or null if unknown`

Reply:
We need to add the ability to get the affinity for any thread in [Java-Thread-Affinity](https://github.com/OpenHFT/Java-Thread-Affinity) I think.

Reply:
I think we can do this change and introduce that functionality later.